lib/routes/nbt/index.ts

```diff
@@ -0,0 +1,73 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import got from '@/utils/got';
+
+const host = 'https://www.nbt.edu.cn/';
+const typeMap = {
+    importance: '/cs_list.jsp?urltype=tree.TreeTempUrl&wbtreeid=1013',
+    news: '/lgxw/mtlg.htm',
+    notice: '/cs_list.jsp?urltype=tree.TreeTempUrl&wbtreeid=1015',
+};
+
+const titleMap = {
+    importance: '理工要闻',
+    news: '媒体理工',
+    notice: '理工快讯',
+};
+
+export const route: Route = {
+    path: '/:type',
+    categories: ['university'],
+    example: '/nbt/importance',
+    parameters: { type: '通知类型，默认为理工要闻' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '理工要问',
+    maintainers: ['bellongyan'],
+    handler,
+    description: `| 理工要闻 | 媒体理工 | 理工快讯 |
+  | ---------- | ----- | ------ |
+  | importance | news  | notice |`,
+};
+
+async function handler(ctx) {
+    const type = ctx.req.param('type') ?? 'notice';
+    const link = host + typeMap[type];
```

Comment:
https://github.com/DIYgod/RSSHub/blob/66fcc61b7ff63221bfda29039dcb9f1078d729fb/lib/routes/nbt/index.ts#L6
https://github.com/DIYgod/RSSHub/blob/66fcc61b7ff63221bfda29039dcb9f1078d729fb/lib/routes/nbt/index.ts#L7-L11
This causes [double slash](https://github.com/DIYgod/RSSHub/actions/runs/10753215436/job/29822393374#step:13:20) `//` in the URL.